dls-controls/malcolmjs
# Feature/method inputs #167

## Description

Added the start of the method inputs and a run button, the input state is stored on the attribute.

## Testing instructions

Add a set up instructions describing how the reviewer should test the code
- Review code
- Check Travis build
- Review changes to test coverage
- run `npm start` and naivgate to http://localhost:3000/gui/PANDA/ and see the `save` method.

## Agile board tracking

connect to #167 
connect to #168
